lib/actions/location.tsx

```diff
@@ -47,9 +48,28 @@ export function getCurrentPosition(
         // On error
         (error) => {
           console.log('error getting current position', error)
-          // FIXME, analyze error code to produce better error message.
-          // See https://developer.mozilla.org/en-US/docs/Web/API/GeolocationPositionError
-          dispatch(receivedPositionError({ error }))
+          // On desktop, after user clicks "Use location" from the location fields,
+          // show an alert and explain if location is blocked.
+          if (!isMobile() && error.code === 1) {
+            window.alert(
```

Comment:
A bit clunky, but I think good enough for now! Could we possibly add a TODO here to move this code into `location-field` and handle this more elgantly

Reply:
I added the TODO in https://github.com/opentripplanner/otp-react-redux/pull/989/commits/933d34a0c8fc1838653a27433651b5e21b92c218.